src/aria2p/client.py

```diff
@@ -187,7 +187,7 @@ def call(self, method, params=None, msg_id=None, insert_secret=True):
 
         if insert_secret and self.secret:
             if method.startswith("aria2."):
-                params.insert(0, self.secret)
+                params.insert(0, 'token:'+self.secret)
```

Comment:
Could you use an f-string instead, for consistency? `f'token:{self.secret}'` 

Also the same change needs to be added in `batch_call` and `multicall2` methods!